app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -760,12 +352,12 @@ describe('NetworkController', () => {
               const promisifiedSendAsync = promisify(provider.sendAsync).bind(
                 provider,
               );
-              const { result: chainIdResult } = await promisifiedSendAsync({
```

Comment:
This is a notable change because in using a fake provider, the Infura middleware no longer returns a static result for `eth_chainId`. In other words we have 100% control over what the provider returns now. So using `eth_chainId` to test the provider object is misleading. We don't even need to use a known RPC method — we just want to make sure that `provider.sendAsync` is called.